### ISSUE TITLE
Add support to follow containers logs with DockerComposeContainer

### ISSUE THREAD

**Issue by mfuksa** [testcontainers/testcontainers-java#158]
When using DockerComposeContainer I would like to follow output logs from my containers that are started by DockerComposeContainer. 

Maybe there is a way how to do it but I have not found it. Using `DockerComposeContainer#followOutput(...)`does not do that.
Thanks.

**Comment by rnorth** [comment]
Hi Miroslav,

Yes, that feature doesn't exist yet. I'm currently doing quite a bit of
rework on docker compose support (#156) so I think it would make sense to
add this when that's all resolved - it should be easier then, as well.

I hope this is ok!
Thanks

Richard
On Wed, 15 Jun 2016 at 20:48, Miroslav Fuksa <notifications@github.com>
wrote:

> When using DockerComposeContainer I would like to follow output logs from
> my containers that are started by DockerComposeContainer.
>
> Maybe there is a way how to do it but I have not found it. Using
> DockerComposeContainer#followOutput(...)does not do that.
> Thanks.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/testcontainers/testcontainers-java/issues/158>, or mute
> the thread
> <https://github.com/notifications/unsubscribe/AAIET8qS7KiblwHbmXr4jCuF31mGoCshks5qL-aQgaJpZM4I2Rd_>
> .
>
-- 
Richard

**Comment by mfuksa** [comment]
Hi Richard,

no problem. I appreciate if it could be added (even later). I can access the logs other ways, so it's fine for me as a workaround. 

Thanks a lot
Miroslav

**Comment by pcornish** [comment]
@mfuksa @rnorth 

Something like this in `DockerComposeContainer.startServices()` might do it:

```
 // tail child container logs
dockerClient.listContainersCmd()
                .withLabelFilter("name=/" + identifier)
                .withShowAll(true)
                .exec().forEach(container -> {
                    followOutput(container.getId(), new Slf4jLogConsumer(logger())
                            .withPrefix(container.getImage()));
});
```

**Comment by outofcoffee** [comment]
@rnorth we can probably close this now #233 has been merged.

**Comment by rnorth** [comment]
Yep - released in 1.1.7 so will close.
